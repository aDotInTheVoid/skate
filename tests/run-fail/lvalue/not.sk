fn main() {
    // TODO: Consider suggesting ==
    2 + 2 = "ds";
}
