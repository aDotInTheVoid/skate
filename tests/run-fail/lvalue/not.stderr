error: Excpected an lvalue
  ┌─ $DD/tests/run-fail/lvalue/not.sk:3:5
  │
3 │     2 + 2 = "ds";
  │     ^^^^^

